docs/asset-store/docs/05-01-asset-store.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Asset Store chart
+type: Configuration
+---
+
+To configure the Asset Store chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+To learn more about how to use overrides in Kyma, read [this document](/root/kyma/#configuration-helm-overrides-for-kyma-installation). See also examples of [top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides) and [sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides).
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **minio.mode** | Specifies Minio mode. | `standalone` |
```

Comment:
add link to doc about switching mode to gateway

Reply:
`mode` parameter is not related to `gateway`. It only defines if MinIO is `distributed` or `standalone`. Please see overrides in the tutorial: https://kyma-project.io/docs/master/components/asset-store/#tutorials-set-minio-to-the-google-cloud-storage-gateway-mode